source/includes/api/order.md

```diff
@@ -362,195 +351,3 @@ curl -X "GET" "https://dashboard.shoprunback.com/api/v1/orders?page=1" \
     ]
 }
 ```
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-## Update an order
```

Comment:
where did this go?

Reply:
The PUT endpoint for orders has been removed